octorun/src/octokit.js

```diff
@@ -0,0 +1,19 @@
+var Octokit = require('octokit-rest-for-node-v0.12');
+
+var createOctokit = function () {
+    return Octokit({
+        timeout: 0,
+        requestMedia: 'application/vnd.github.v3+json',
+        headers: {
+            'user-agent': 'octokit/rest.js v1.2.3'
```

Comment:
This needs to be the same string that we pass in C# as ProductHeaderValue - the name and version of the app. It should probably be set in the environment by C#, probably easiest that way.

Reply:
So I'm looking at what value is being used currently...

https://github.com/github-for-unity/Unity/blob/b3850c6d6206eef04fbbeeb3cb2d1d1fa940a0ab/src/GitHub.Api/Application/ApplicationConfiguration.cs#L14

https://github.com/github-for-unity/Unity/blob/b3850c6d6206eef04fbbeeb3cb2d1d1fa940a0ab/src/GitHub.Api/Application/ApplicationInfo.cs#L6-L14

We are using `ApplicationInfo.ApplicationSafeName`, should it not be? `ApplicationInfo.ApplicationName`

Reply:
Yeah, it should, dunno why safe name is there like that.

Reply:
Actually it should always be `GitHubUnity` so you can just set `ApplicationSafeName` to that

Reply:
Tracked in #620